walletnode/services/externaldupedetection/config.go

```diff
@@ -0,0 +1,68 @@
+package externaldupedetetion
+
+import (
+	"time"
+
+	"github.com/pastelnetwork/gonode/walletnode/services/common"
+)
+
+const (
+	defaultNumberSuperNodes = 3
+
+	defaultNumberRQIDSFiles uint32 = 10
+
+	defaultConnectToNextNodeDelay = time.Millisecond * 200
+	defaultAcceptNodesTimeout     = defaultConnectToNextNodeDelay * 10 // waiting 2 seconds (10 supernodes) for secondary nodes to be accpeted by primary nodes.
```

Comment:
Please increase it to 30s, refer https://github.com/pastelnetwork/gonode/commit/d7c097f0c4ae24cb2da10643e5d2c5303e3d976b